src/app/admin/services/api/api.service.ts

```diff
@@ -39,14 +39,18 @@ export class ApiService {
     });
   }
 
-  blockEquipment(id: number, period: Period) {
+  blockEquipment(id: number, period: Period): Observable<unknown> {
     const body = {
       end_date: period.endDate,
       start_date: period.startDate,
     };
     return this.http.post<string>(`/equipment/${id}/blocking`, body);
   }
 
+  unblockEquipment(id: number): Observable<null> {
```

Comment:
I think void observable is more preferrable to indicate that code doesn't rely on the emitted value and null observable is rarely used.